### PR TITLE
Remove startup stuff like countdown and teams info

### PR DESCRIPTION
**Pull request by TurBoss**

### CODE REVIEW

File: LuaRules/Gadgets/gui_settings.lua [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		author	= 'TurBoss',
+		version	= 'v0.1',
+		date	= 'Oct 25-10-2015 ',
+		license	= 'GNU GPL, v2 or later',
+		layer	= 0,
+		enabled	= true
+	}
+end
+
+if (gadgetHandler:IsSyncedCode()) then
```
Comment (gajop) [inline LuaRules/Gadgets/gui_settings.lua]: () is not needed around if/for/when in Lua

File: LuaRules/Gadgets/gui_settings.lua [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	return
+end
+
+function gadget:GameSetup(state,ready,playerStates)
```
Comment (gajop) [inline LuaRules/Gadgets/gui_settings.lua]: if params aren't used there's no need to list them; i prefer to have function defs minimal, thus :GameSetup() in this case

**Comment by TurBoss** [comment]
ok

**Comment by TurBoss** [comment]
Done also added the little loadbar style

**Comment by gajop** [comment]
Cool, Thanks!

**Pull request merged**